src/acsets/acsets.py

```diff
@@ -102,22 +93,19 @@ class AttrType(HashableBaseModel):
     transition, for instance, has a tuple of strings as its name.
     """
 
-    name: str
-    ty: type
-
-    def __init__(self, name: str, ty: type) -> None:
-        """Initialize a new attribute type for a schema.
-
-        Args:
-            name: The name of the attribute type.
-            ty: The type assigned to the attribute type.
-        """
-        super(AttrType, self).__init__(name=name, ty=ty)
+    name: str = Field(description="The name of the attribute type.")
+    ty: type = Field(description="The type assigned to the attribute type.")
```

Comment:
@olynch can you comment on this? Do we need to have this type be defined here, or can we just use a string representation?

Reply:
How did you end up addressing this?

Reply:
See https://github.com/AlgebraicJulia/py-acsets/pull/15/commits/f1585a3f844d91a4a9b25a8a2a4ac340471ebf60

Reply:
We can plan a call and I can code walk you through it on Monday if you have time before 10:30 est